client_test.go

```diff
@@ -416,10 +412,11 @@ func TestUpdate(t *testing.T) {
 	// The builder should be invoked with a service name and path to its source
 	// function code.  For this test, it is a name derived from the test path.
 	// An example image name is returned.
-	builder.BuildFn = func(name2, path2 string) (string, error) {
-		if name != name2 {
-			t.Fatalf("builder expected name %v, got '%v'", name, name2)
-		}
+	// TODO: FIX ME UNDERSTAND ME
+	builder.BuildFn = func(tag string) (string, error) {
+		// if name != name2 {
+		// 	t.Fatalf("builder expected name %v, got '%v'", name, name2)
+		// }
```

Comment:
should this perhaps be `if tag != expectedTag`?  It's a simple logical passthrough at this point, so the test doesn't add a whole lot of value, merely level-setting for ensuring that future complexity doesn't break the base case, but it may be worthwhile to keep plumbed.

Reply:
Ack - I just missed this when going through the tests.